asv_bench/benchmarks/categoricals.py

```diff
@@ -8,4 +8,18 @@ def setup(self):
         self.s = pd.Series((list('aabbcd') * 1000000)).astype('category')
 
     def time_concat_categorical(self):
-        concat([self.s, self.s])
\ No newline at end of file
+        concat([self.s, self.s])
+
+
+class categorical_value_counts(object):
+    goal_time = 1
+
+    def setup(self):
+        n = 500000
+        np.random.seed(2718281)
+        arr = ['s%04d' % i for i in np.random.randint(0, n // 10, size=n)]
+        self.ts = Series(arr).astype('category')
+
+    def time_value_counts(self):
+        self.ts.value_counts(dropna=True)
+        self.ts.value_counts(dropna=False)
```

Comment:
You get a timing per function. So if you want to track performance of both with dropna True and False, it has to be in two functions.